### DISCUSSION TITLE
Cannot build projects after updated the Uno Platform VS Extension.

### DISCUSSION THREAD

**Discussion by LITTOMA** [unoplatform/uno#14933] [Q&A]
Error logs:
```
XamlCompiler error WMC1006: Cannot resolve Assembly or Windows Metadata file 'Type universe cannot resolve assembly: Uno.UI, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null.'
```

**Comment by limeniye** [comment]
Hello, @LITTOMA.
For me all is works well. Can you attach a sample?

**Reply by LITTOMA** [reply]
I don't know how. A new project can be built with no errors. However my old project created by previous Uno extension can't be built. It's a company project so I can't provide source code.

**Reply by jeromelaban** [reply]
In general, this means that you have an uno package dependency in your `.Windows` head. Try commenting out packages one by one to see which one causes this.
